### DISCUSSION TITLE
How to join tables across different postgresql databases on different severs?

### DISCUSSION THREAD

**Discussion by jrmidkiff** [sqlalchemy/sqlalchemy#10244] [Usage Questions] [answered]
I have two different postgresql databases on two different servers due to a migration process, and I would like to compare the tables between those different databases to ensure the migration is correctly moving all data as planned. Is that possible with SQLAlchemy?

**Comment by CaselIT** [comment]
Hi,

Do you have a specific sql syntax in mind that you want to render using sqlalchemy or are you asking if there is a way of doing it in SQL (pg specific)?

**Reply by jrmidkiff** [reply]
I don't have specific syntax in mind, and was just wondering if it's something SQLAlchemy is even capable of doing

**Reply by CaselIT** [reply, marked as answer]
This is more something that the db needs to be capable of doing.

I'm no expert on this part of postgresql, but you can take a look at https://www.postgresql.org/docs/current/postgres-fdw.html

The general idea would be to connect one db to the other using that extension. Sqlalchemy only connect to the first db (the one that can access tables of the remote db), and you can then use the normal features to query tables of both dbs

**Reply by jrmidkiff** [reply]
Ah, that makes sense. Thank you!!
